hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/ConfigurationKeys.java

```diff
@@ -59,6 +59,9 @@
   public static final String FS_AZURE_ENABLE_AUTOTHROTTLING = "fs.azure.enable.autothrottling";
   public static final String FS_AZURE_ALWAYS_USE_HTTPS = "fs.azure.always.use.https";
   public static final String FS_AZURE_ATOMIC_RENAME_KEY = "fs.azure.atomic.rename.key";
+  /** Provides a config to provide comma separated path prefixes on which Appendblob based files are created
+   *  Default is empty. **/
+  public static final String FS_AZURE_APPEND_BLOB_KEY = "fs.azure.appendblob.key";
```

Comment:
fs.azure.appendblob.key config name doesn't look good. Wouldsomething like fs.azure.appendblob.directories convey the meaning better?

Reply:
I have made is similar to FS_AZURE_ATOMIC_RENAME_KEY which also provide a set of directories. Further please note for appendblob, this is actually a prefix for the path (and not necessarily the directories). This is done so that the test suite (which all runs on a container with a randon guid can run) can run on appendblob based files. Let me know ur comments/thoughts here.

Reply:
done